app/environments/brassbirmingham/brassbirmingham/envs/brassbirmingham.py

```diff
@@ -0,0 +1,384 @@
+
+import gym
+import numpy as np
+import random
+
+import config
+
+from stable_baselines import logger
+
+from .classes import *
+
+class BrassBirminghamEnv(gym.Env):
+    metadata = {'render.modes': ['human']}
+
+    def __init__(self, verbose = False, manual = False):
+        super(BrassBirminghamEnv, self).__init__()
+        self.name = 'brassbirmingham'
+        self.n_players = 3
+
+        self.manual = manual
+
+        self.board_size = 7
+        self.squares = self.board_size * self.board_size
+
+        self.tile_types = 11
+        
+        self.max_score = 100
+        
+        self.total_positions = self.squares + self.n_players + 1
+
+        self.set_contents()
+        self.nets = [5,7,16, 24, 32, 41, 43]
+        self.total_tiles = sum([x['count'] for x in self.contents])
+
+        self.action_space = gym.spaces.Discrete(self.total_tiles  * 2)
+        self.observation_space = gym.spaces.Box(0, 1, (self.total_positions * self.total_tiles + self.squares + 4 + self.n_players + self.action_space.n ,))
+        self.verbose = verbose
```

Comment:
I think basically everything related to that `Butterfly` game that isn't relevant to BB should be removed to prevent future confusion. And we can just add TODOs